sw/device/silicon_creator/manuf/base/BUILD

```diff
@@ -265,6 +265,7 @@ cc_library(
         "//sw/device/silicon_creator/lib/cert:tpm_ek_template_library",
         "//sw/device/silicon_creator/lib/drivers:flash_ctrl",
         "//sw/device/silicon_creator/lib/drivers:hmac",
+        "//sw/device/silicon_creator/manuf/lib:flash_info_fields",
```

Comment:
you don't need this dep. you only need the `"//sw/device/lib/dif:flash_ctrl",` which is already included above.